dev/breeze/SELECTIVE_CHECKS.md

```diff
@@ -141,7 +141,7 @@ The selective check outputs available are described below:
 | needs-helm-tests                   | Whether Helm tests are needed to run ("true"/"false")                                                  | true                                                          |
 | needs-javascript-scans             | Whether javascript CodeQL scans should be run ("true"/"false")                                         | true                                                          |
 | needs-python-scans                 | Whether Python CodeQL scans should be run ("true"/"false")                                             | true                                                          |
-| postgres-exclude                   | Which versions of Postgres to exclude for tests as JSON array                                          | []                                                            |
+| postgres-exclude                   | Which versions of Postgres to exclude for tests as JSON array                                          | ['3.8']                                                       |
```

Comment:
This also needs to be done here to actually work:
https://github.com/apache/airflow/blob/8f5e100f30764e7b1818a336feaa8bb390cbb327/dev/breeze/src/airflow_breeze/utils/selective_checks.py#L356

Reply:
yep - what Jed wrote is what's needed.